dev-util/hip/hip-5.1.3-r1.ebuild

```diff
@@ -23,7 +25,7 @@ KEYWORDS="~amd64"
 LICENSE="MIT"
 SLOT="0/$(ver_cut 1-2)"
 
-IUSE="debug profile"
+IUSE="debug doc profile"
```

Comment:
the doc flag is already set by `docs.eclass`

Reply:
Got it